File: Beecrowd/1015 Distância Entre Dois Pontos/main.py
import math

entrada = [float(i) for i in input().split()]
x1 = entrada[0]
y1 = entrada[1]

entrada1 = [float(i) for i in input().split()]
x2 = entrada1[0]
y2 = entrada1[1]


Distancia = math.sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2))
print("%0.4lf" %Distancia)

File: Beecrowd/1015 Distância Entre Dois Pontos/main.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main() {

    double x1, x2, y1, y2, Distancia;

    cin >> x1;
    cin >> y1;
    cin >> x2;
    cin >> y2;

    Distancia = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));

    cout << fixed;
    cout.precision(4);
    cout << Distancia << endl;

    return 0;
}